File: JAVA/Exercicio001.java
package main;

import java.util.Scanner;

public class Exercicio001 {
	
	public static int NUM1 = 1;
	public static int NUM2 = 2;
	public static int SOMA = 0;

	public static void main(String[] args) {
		
		System.out.println("Entre com dois valores do tipo Inteiro:");
		
		try (Scanner teclado = new Scanner(System.in)) {
			NUM1 = teclado.nextInt();
			NUM2 = teclado.nextInt();
			teclado.close();
		}
		SOMA = NUM1 + NUM2;
		
		System.out.println("A soma dos inteiros = " + SOMA);
	}

}

File: PHP/Exercicio001.php
<!DOCTYPE html>
<html>

<body>

  <form method="post">
    Entre com o primeiro Inteiro:</br>
    <input type="text" id="NUM1" name="NUM1" required></br>
    Entre com o segundo Inteiro:</br>
    <input type="text" id="NUM2" name="NUM2" required></br>   
    <button>Somar</button></br>
  </form>
<?php

  $NUM1 = $_POST['NUM1'];
  $NUM2 = $_POST['NUM2'];
  $SOMA = 0;

  $SOMA = $NUM1 + $NUM2;
  echo "A soma dos inteiros eh: " . $SOMA;
?>
</body>
</html>